jbpm-workitem-itests/src/test/java/org/jbpm/workitem/springboot/samples/KafkaFixture.java

```diff
@@ -196,4 +199,12 @@ protected static boolean isDockerAvailable() {
             return false;
         }
     }
+    
+    protected static String getVersion(String version) {
+        Matcher matcher = Pattern.compile("(\\d+\\.\\d+\\.\\d+)").matcher(version);
```

Comment:
minor - maybe it's worth checking that `version` param is not null and make it more robust? wdyt?